src/components/Chat.js

```diff
@@ -1,3 +1,7 @@
+//useRef, io, link, edit, make_room, chat_bubble, and rocket are defined but never used
```

Comment:
`useRef` can be safely deleted from line 1 of the original file, line 5 of the current commit.